src/content/app/App.tsx

```diff
@@ -41,47 +40,52 @@ const AppContainer = () => {
     };
   }, [location.pathname]);
 
+  useLocationReporting();
+
   return <App />;
 };
 
+const useLocationReporting = () => {
+  const { pathname, search, hash } = useLocation();
+
+  useEffect(() => {
+    const pathToReport = `${pathname}${search}${hash}`;
+    analyticsTracking.trackPageView(pathToReport);
+  }, [pathname]);
+};
+
 const App = memo(() => {
+  const routes = useRoutes(routesConfig);
+
   return (
     <>
       <Header />
-      <Switch>
-        {routes.map((route, index) => (
-          <Route
-            key={index}
-            path={route.path}
-            exact={route.exact}
-            render={(props) => <route.component {...props} />}
-          />
-        ))}
-        <Route component={NotFound} />
-      </Switch>
+      {routes}
+      {/*  remember to handle the 404's (update commented code below) */}
```

Comment:
I guess this is for this PR itself?

Reply:
Yeah, the 404s are being handled; I'll need to clean up this file.